test/test_formatter.js

```diff
@@ -204,6 +204,65 @@ Picture: http://example.com/security-camera.jpg`
     }, null,
     [ 'The total file size is 7.' ]
     ],
+
+    ['com.wikicfp:search', {
+        start: new Date('TBD'),
+        end: new Date('TBD'),
```

Comment:
By the way, passing arbitrary stuff in the Date constructor is risky. I think the implementation of wikicfp should detect TBD and pass a null/NaN explicitly.